install/install.md

```diff
@@ -10,23 +10,91 @@ description: |
 # RAPIDS Installation Guide
 {: .fs-8 }
 
-RAPIDS has several methods for installation, depending on the preferred environment and versioning. Returning users can jump straight to our **[release selector](#selector)** and new users can get started with these four steps:
+RAPIDS has several methods for installation, depending on the preferred environment and versioning. **New Users** should be mindful of the system and environment prerequisites. 
 {: .fs-6 .fw-300 }
 
-**[1. Provision System](#provision)** <br/>
-**[2A. Set Up Environment](#set-up)** <br/>
-**[2B. Set Up WSL2 Environment](#WSL2)** <br/>
-**[3A. Install RAPIDS (Release Selector)](#selector)** <br/>
-**[3B. Install RAPIDS (pip)](#pip-install)** <br/>
-**[4. Getting Started](#start)**  
+**[Install RAPIDS with Release Selector](#selector)** <br/>
+- [Installation Troubleshooting](#troubleshooting)
+
+**[System Requirements](#system-req)** <br/>
+- [OS, NVIDIA GPU Driver, and CUDA Versions](#system-req)
+- [(Optional) System Recommendations](#system-recommendations)
+- [(Optional) Cloud Instance GPUs](#cloud-gpu)
+
+**[Environment Setup](#environment)** <br/>
+- [Conda](#conda)
+- [Docker](#docker)
+- [Pip](#pip)
```

Comment:
I vote lowercase always if possible. I have worked with other software packages that use the "lowercase always" rule. It can be awkward but you quickly learn how to avoid starting sentences with that word, which lets you avoid the problem entirely.

Reply:
As far as I can tell the pip documentation always has it lower case - so Ill follow that.